cypress/integration/score_card_spec.js

```diff
@@ -0,0 +1,23 @@
+describe('Scorecard user flow', () => {
+
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+      .get('.search-btn').click().click()
```

Comment:
I don't know why we have to click twice here. It was doing the same thing yesterday.